src/store/configureStore.js

```diff
@@ -0,0 +1,46 @@
+/* global __DEVELOPMENT__ */
+import { compose, createStore, applyMiddleware } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+import { createLogger } from 'redux-logger'
+import flag from 'cozy-flags'
```

Comment:
Are we using this?

Reply:
Not yet! But it should be added in our skeleton :D